package/MDAnalysis/selections/gromacs.py

```diff
@@ -34,7 +34,7 @@
 The index groups are named *mdanalysis001*, *mdanalysis002*, etc.
 
 .. _Gromacs: http://www.gromacs.org
-.. _ndx: http://www.gromacs.org/Documentation/File_Formats/Index_File
+.. _ndx: https://manual.gromacs.org/2024.4/reference-manual/file-formats.html#ndx
```

Comment:
The other GMX links uses a slightly different version -- 2024.0. Could we use the same version for all of them?